spec/system/sdg_management/relations_spec.rb

```diff
@@ -407,6 +407,14 @@
           expect(find("input[data-code='1']")).to be_checked
         end
       end
+
+      scenario "Help page link opens in new window" do
+        process = create(:legislation_process, title: "SDG process")
+
+        visit sdg_management_edit_legislation_process_path(process)
+
+        expect(find_link("help")[:target]).to eq("_blank")
```

Comment:
How about testing it from the user's point of view?

```ruby
        within_window(window_opened_by { click_link "SDG help page" }) do
          expect(page).to have_content "Sustainable Development Goals help"
        end
```
Writing tests from the user's point of view in this case made me realize that maybe the "help" text in the link is not specific enough, so I added a small change:

```diff
diff --git a/config/locales/en/sdg.yml b/config/locales/en/sdg.yml
index 36ac548062..06b7559e65 100644
--- a/config/locales/en/sdg.yml
+++ b/config/locales/en/sdg.yml
@@ -461,8 +461,8 @@ en:
       goal_list: "You can choose one or several SDGs aligned with your %{record}"
       help:
         title: "Which SDGs and targets are aligned with my %{record}?"
-        text: "help"
-      hint: You can introduce the code of a specific goal/target or a text to find one. For more information visit the %{link} page.
+        text: "SDG help page"
+      hint: You can introduce the code of a specific goal/target or a text to find one. For more information visit the %{link}.
       label: You can introduce the code of a specific goal/target or a text to find one
       placeholder: "Write a goal or target code or description"
       remove_tag: "Remove"
diff --git a/config/locales/es/sdg.yml b/config/locales/es/sdg.yml
index 304a4d8af4..8945e4acb3 100644
--- a/config/locales/es/sdg.yml
+++ b/config/locales/es/sdg.yml
@@ -461,8 +461,8 @@ es:
       goal_list: "Puedes seleccionar uno o varios ODS con los que se alinea tu %{record}"
       help:
         title: "¿Qué ODS y metas se alinean con mi %{record}?"
-        text: "ayuda"
-      hint: Puedes introducir el código de un objetivo/meta específico o un texto para encontrar uno. Para más información visita la página de %{link}.
+        text: "página de ayuda de ODS"
+      hint: Puedes introducir el código de un objetivo/meta específico o un texto para encontrar uno. Para más información visita la %{link}.
       label: Puedes introducir el código de un objetivo/meta específico o un texto para encontrar uno
       placeholder: "Escribe las etiquetas que desees"
       remove_tag: "Eliminar"
```